config.json

```diff
@@ -2225,8 +2249,8 @@
       {
         "groupId": "com.google.guava",
         "artifactId": "guava",
-        "version": "33.1.0-android",
-        "nugetVersion": "33.1.0.1",
+        "version": "33.2.0-android",
+        "nugetVersion": "33.2.0-android",
```

Comment:
This one we manually remove `-android` from because it is not a preview package.

Reply:
I know, but I am afraid this is not good idea. There is more and more library vendors that use `-jre` versions with latest Android SDKs and modern java. Nuget will not be able to pick correct versions, so we might end up with subtle runtime errors `java.lang.NoSuchMethodError` which will cause us more digging, debugging, analyzing - effort.

Normally I would have to:

1. check bindings if the method is there
2. check optimizations trimming/linking and proguard
3. dive in further which might be highly time consuming (for example we remove version during download of artifact)

The problem is with Maven and Gradle and we are trying to fix other build systems and violating/misuse of semantic versioning for metadata by some library authors and they don't even see it as problem.